### PR TITLE
Gh 235 user selection by javascript

### PR DESCRIPTION
**Pull request by jwacalex**
frist draft to fix issue #235. needs styling for dark themes

### CODE REVIEW

File: includes/pages/user_shifts.php [hunk 1]
```diff
@@ -167,7 +167,7 @@ function view_user_shifts() {
   return page([
       div('col-md-12', [
           msg(),
-          template_render('../templates/user_shifts.html', [
+          template_render('../templates/user_shtifts.html', [
```
Comment (MyIgel) [inline includes/pages/user_shifts.php]: ?

File: .gitmodules [hunk 2]
```diff
@@ -4,3 +4,6 @@
 [submodule "vendor/bootstrap"]
 	path = vendor/bootstrap
 	url = https://github.com/twbs/bootstrap.git
+[submodule "vendor/select2"]
```
Comment (MyIgel) [inline .gitmodules]: I removed the submodules in feature-igel-rewrite / b839e401062b294292fdcbd7e30b79bc149fab6f so we should load the [select2/select2](https://packagist.org/packages/select2/select2) via composer

**Comment by jwacalex** [comment]
fully agree with your suggestions. this was a quick and dirty approach to have a working proof of concept. if you want to integrate it in your rewrite i would suggest to close this request

**Comment by msquare** [comment]
I will close the request as suggested.

**Pull request closed without merging**